### PR TITLE
Python3 migration

### PR DESCRIPTION
**Pull request by tylerjw**
### Description

This makes the move_group_python_interface_tutorial both python2 and python3 compatible.  This is needed to run this tutorial on Noetic.

### CODE REVIEW

**Review by rhaschke — CHANGES_REQUESTED** [review]
Thanks for starting this. There are other not yet python3 compliant python files though. Please handle them all:
doc/collision_environments/scripts/collision_scene_example.py
_scripts/tutorialformatter.py

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py [hunk 1]
```diff
@@ -47,6 +47,7 @@
 import moveit_commander
 import moveit_msgs.msg
 import geometry_msgs.msg
+from builtins import input
```
[the remark this answers is not in the document]
Comment (tylerjw) [reply doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: done

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py [hunk 2]
```diff
@@ -119,21 +120,21 @@ def __init__(self):
     ## ^^^^^^^^^^^^^^^^^^^^^^^^^
     # We can get the name of the reference frame for this robot:
     planning_frame = move_group.get_planning_frame()
-    print "============ Planning frame: %s" % planning_frame
+    print("============ Planning frame: %s" % planning_frame)
```
[the remark this answers is not in the document]
Comment (tylerjw) [reply doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py:126]: done

**Comment by tylerjw** [comment]
> Thanks for starting this. There are other not yet python3 compliant python files though. Please handle them all:
> doc/collision_environments/scripts/collision_scene_example.py
> _scripts/tutorialformatter.py

@rhaschke I updated this to include changes for collision_scene_example.py but was not able to find anything wrong with tutorialformatter.py.  I also don't understand how tutorialformatter.py is used so I wasn't able to test it outside of just calling it with python3 (which runs without error and produces no output).

**Comment by rhaschke** [comment]
> I was not able to find anything wrong with tutorialformatter.py.

Sorry for the noise. I was looking at an older version of the file. Michael already fixed some stuff a while ago (#410).

> I also don't understand how tutorialformatter.py is used so I wasn't able to test it outside of just calling it with python3

It's used as a plugin to sphinx, generating the docs:
https://github.com/ros-planning/moveit_tutorials/blob/f0ac33f91603d7bffa8592d9a9b4943cec7eedea/conf.py#L5-L6

**Review by rhaschke — APPROVED** [review]

**Pull request merged**